cmd/hubagent/workload/setup.go

```diff
@@ -93,32 +99,32 @@ func SetupControllers(ctx context.Context, wg *sync.WaitGroup, mgr ctrl.Manager,
 
 	discoverClient := discovery.NewDiscoveryClientForConfigOrDie(config)
 	// Verify CRD installation status.
-	if opts.EnableV1Alpha1APIs {
-		for _, gvk := range v1Alpha1RequiredGVKs {
+	if opts.EnableClusterInventoryAPIs {
```

Comment:
to enable the clusterInventory API, we also need v1beta1 MC API.

Move to the v1beta1 checking block?